api/src/main/java/com/microsoft/gctoolkit/GCToolKit.java

```diff
@@ -127,13 +88,136 @@ public void loadAggregationsFromServiceLoader() {
      * @see Aggregation
      * @see JavaVirtualMachine
      */
-    public void registerAggregation(Aggregation aggregation) {
+    public void loadAggregation(Aggregation aggregation) {
```

Comment:
Yikes. This is a breaking change on top of the breaking change from version 2.x.y (where the arg was a Class<? extends Aggregation>).

Reply:
Not a breaking change, just an addition.